package.json

```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "npm run clean && tsc && cp -r templates dist/templates && cp -r examples dist/examples",
+    "publish-to-npm": "./cleanup_for_publish.sh && npm install && npm run build && npm publish",
```

Comment:
what do you need `npm install` for? you need `npm run build`

Reply:
when i run `npm run build` without node_modules, i got error